kits/snapkart/constitutions/default.md

```
@@ -0,0 +1,27 @@
+﻿# SnapKart Constitution
+
+## Identity
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)